src/main/java/com/webapp/TextBook/repository/shared_respository_helper/DataAccessConversionHelper.java

```diff
@@ -0,0 +1,44 @@
+package com.webapp.TextBook.repository.shared_respository_helper;
+
+import com.webapp.TextBook.repository.DataAccessConversion;
+import com.webapp.TextBook.repository.DataAccessConversionException;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.function.Supplier;
+
+
+public class DataAccessConversionHelper {
+
```

Comment:
Looks like 'T' represents a DataAccessConversion type. It also looks shared across all methods in this ORM-DBO helper. Why can't we place it as a class type param? That seems like a good idea!